src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -180,6 +192,23 @@ class TargetStorage
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+public:
+    CHIP_ERROR Serialize(chip::TLV::TLVWriter & writer)
+    {
+        ReturnErrorOnFailure(writer.Put(chip::TLV::AnonymousTag(), mCluster));
+        return writer.Put(chip::TLV::AnonymousTag(), mDeviceType);
+    }
+
+    CHIP_ERROR Deserialize(chip::TLV::TLVReader & reader)
+    {
+        // mCluster
+        ReturnErrorOnFailure(reader.Next(chip::TLV::AnonymousTag()));
+        ReturnErrorOnFailure(reader.Get(mCluster));
+        // mFabricIndex
```

Comment:
what do these comments mean?

Above I see mCluster and then a get for mCluster ... in which case I would say the comment is probably redundant.

But then here I see a comment for mFabricIndex and a get for mDeviceType.

Could you re-phrase things  like `TODO: mFabricIndex is not yet here` or on top `Format: cluster, fabric, device` (in which case this read is not clearly correct)?

Reply:
Since we have serialize above, probably just delete the comments is sufficient. At most if we ever need fabric index, we can add a TODO (also if that is the case, how will we be backwards compatible? I imagine we have to add fabric index at the end).

Reply:
These were copied from another use of TLV encoding. I've removed them since they're explained by the reader calls below, and could be misleading (as in this case; the second should've been mDeviceType).